route_test.go

```diff
@@ -1997,18 +1996,26 @@ func TestSEG6LocalEqual(t *testing.T) {
 	}
 }
 func TestSEG6RouteAddDel(t *testing.T) {
-	if os.Getenv("CI") == "true" {
-		t.Skipf("Fails in CI with: route_test.go:*: Invalid Type. SEG6_IPTUN_MODE_INLINE routes not added properly")
-	}
-	// add/del routes with LWTUNNEL_SEG6 to/from loopback interface.
+	// add/del routes with LWTUNNEL_SEG6 to/from interface.
 	// Test both seg6 modes: encap (IPv4) & inline (IPv6).
 	t.Cleanup(setUpSEG6NetlinkTest(t))
 
-	// get loopback interface and bring it up
-	link, err := LinkByName("lo")
+	// loopback doesn't work on recent kernels, so use a dummy
+	err := LinkAdd(&Dummy{LinkAttrs{Name: "eth0"}})
```

Comment:
Should we give the dummy device a name which is unlikely to conflict with a real device name on the host?
People can run go test on their linux VM where `eth0` may exist.

Reply:
This is in a brand new isolated network namespace so there should be no conflict. I used eth0 because some of the other tests use that name.